python/paddle/nn/layer/pooling.py

```diff
@@ -321,6 +322,211 @@ def extra_repr(self):
+            The default value is 0.
+        ceil_mode(bool, optional): ${ceil_mode_comment}Whether to use the ceil function to calculate output height
+            and width. If it is set to False, the floor function will be used. The default value is False.
+        data_format(str, optional): The data format of the input and output data. An optional string from: `"NCL"`,
+            `"NLC"`. The default is `"NCL"`. When it is `"NCL"`, the data is stored in the order of:
+            `[batch_size, input_channels, input_length]`.
+        name(str, optional): For eed to detailed information, please refer to :ref:`api_guide_Name`.
+            Usually name is no nset and None by default.
+
+    Shape:
+        - x(Tensor): The input tensor of lp pool1d operator, which is a 3-D tensor.
+          The data type can be float32, float64.
+        - output(Tensor): The output tensor of lp pool1d  operator, which is a 3-D tensor.
+          The data type is same as input x.
+
+    Returns:
+        A callable object of LPPool1D.
+
+    Examples:
+
+        .. code-block:: python
+
+            >>> import paddle
+            >>> import paddle.nn as nn
+
+            >>> data = paddle.uniform([1, 3, 32], dtype="float32", min=-1, max=1)
+            >>> LPPool1D = nn.LPPool1D(norm_type=2, kernel_size=2, stride=2, padding=0)
+            >>> pool_out = LPPool1D(data)
+            >>> print(pool_out.shape)
+            [1, 3, 16]
+
+    """
+
+    def __init__(
+        self,
+        norm_type,
+        kernel_size,
+        stride=None,
+        padding=0,
+        ceil_mode=False,
+        data_format="NCL",
+        name=None,
+    ):
+        super().__init__()
+        self.norm_type = float(norm_type)
+        self.kernel_size = kernel_size
+        self.stride = stride
+        self.padding = padding
+        self.ceil_mode = ceil_mode
+        self.data_format = data_format
+        self.name = name
+
+    def forward(self, x):
+        out = F.lp_pool1d(
+            x,
+            self.norm_type,
+            self.kernel_size,
+            self.stride,
+            self.padding,
+            self.ceil_mode,
+            self.data_format,
+            self.name,
+        )
+        return out
+
+    def extra_repr(self):
+        return 'norm_type={norm_type}, kernel_size={kernel_size}, stride={stride}, padding={padding}'.format(
+            **self.__dict__
+        )
+
+
+class LPPool2D(Layer):
+    r"""
+    This operation applies 2D power-average pooling over input features based on the input,
+    and kernel_size, stride, padding parameters. Input(X) and Output(Out) are
+    in NCHW format, where N is batch size, C is the number of channels,
+    H is the height of the feature, and W is the width of the feature.
+
+    Example:
+        Input:
+            X shape: :math:`(N, C, :math:`H_{in}`, :math:`W_{in}`)`
```

Comment:
内部的 :math: 标记重叠了，这会导致解析错误
```suggestion
            X shape: :math:`(N, C, H_{in}, W_{in})`
```